client/reader/recent/index.tsx

```diff
@@ -75,10 +79,12 @@ const Recent = () => {
 	const fields = useMemo(
 		() => [
 			{
-				id: 'seen',
-				label: translate( 'Seen' ),
+				id: 'icon',
+				label: translate( 'Icon' ),
 				render: ( { item }: { item: ReaderPost } ) => {
-					return <RecentSeenField post={ getPostFromItem( item ) } />;
+					const post = getPostFromItem( item );
```

Comment:
I removed the seen field and replaced it with an icon field that renders the icon in the DataViews `mediaField`.